test/common/router/config_impl_test.cc

```diff
@@ -95,12 +113,25 @@ Http::TestRequestHeaderMapImpl genPathlessHeaders(const std::string& host,
                                         {"x-forwarded-proto", "http"}};
 }
 
+Http::TestRequestHeaderMapImpl genHeaders(const std::string& host, const std::string& path,
+                                          const std::string& method,
+                                          const std::string& forwarded_proto) {
+  auto hdrs = Http::TestRequestHeaderMapImpl{
```

Comment:
nit: avoid abbreviations in variable names

Reply:
This is a standard envoy header name